sdk/src/main/java/openconsensus/sdk/stats/View.java

```diff
@@ -23,7 +23,7 @@
 import javax.annotation.concurrent.Immutable;
 import openconsensus.internal.StringUtils;
 import openconsensus.internal.Utils;
-import openconsensus.stats.Measure;
```

Comment:
so you kept tests namespace for now?

Reply:
This is in the SDK, I did not change that for the moment.